### PR TITLE
add credit scorecard tutorial

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: doc/tutorial/score_card.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Credit ScoreCard Model on SQLFlow
+
+The credit scorecard model is widely used in financial lending scenario, the model outputs a score that represents
```
Comment (sneaxiy) [inline doc/tutorial/score_card.md]: `, the model` -> `. The model`

File: doc/tutorial/score_card.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+how likely the lender will be repaid on time. This tutorial introduces how to train a credit scorecard model with SQLFlow.
+
+## Dataset Introduction
+
+This tutorial using a public dataset [Give Me Some Credit](https://www.kaggle.com/c/GiveMeSomeCredit/data)
+on the [Kaggle](https://www.kaggle.com) website. The dataset contains 150,000 rows training data,
+each row includes 11 columns, where `serious_dlqin2yrs` is the target column, represents person experienced 90 days
```
Comment (sneaxiy) [inline doc/tutorial/score_card.md]: `, each row` -> `, and each row`.

`... is the target column, represents` -> `... is the target column and represents`.

File: doc/tutorial/score_card.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+past due delinquency or worse. The description for each column as the table:
```
Comment (sneaxiy) [inline doc/tutorial/score_card.md]: `The description for each column as the table` -> `The description for each column is as the table`

File: doc/tutorial/score_card.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+Run the following command to start the credit scorecard modeling.
+
+```sql
+SELECT * FROM scorecard.train
+TO TRAIN sqlflow_models.ScoreCard
+LABEL serious_dlqin2yrs
+INTO sqlflow_models.my_scorecard_model;
+```
+
+The above `TRAIN` clause would outputs the scorecard as the following, the final total score
```
Comment (sneaxiy) [inline doc/tutorial/score_card.md]: `, the final` -> `. The final`

File: doc/tutorial/scripts/give_me_some_credit.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import pandas as pd  # data processing, CSV file I/O (e.g. pd.read_csv)
+
+train_df = pd.read_csv("./GiveMeSomeCredit/cs-training.csv", index_col=0)
+
+train_df.MonthlyIncome.fillna(train_df.MonthlyIncome.median(), inplace=True)
+train_df.NumberOfDependents.fillna(train_df.NumberOfDependents.median(),
+                                   inplace=True)
+train_df = train_df[(train_df['age'] > 0) & (train_df['age'] < 90)]
+
+train_df[:2000].to_csv('/tmp/train.csv', index=False)
```
Comment (sneaxiy) [inline doc/tutorial/scripts/give_me_some_credit.py:24]: Why `train_df[:2000]` and why `train_df[(train_df['age'] > 0) & (train_df['age'] < 90)]`? Is it used to limit the data number?

Comment (Yancey0623) [reply doc/tutorial/scripts/give_me_some_credit.py:24]: `train_df[:2000]  ` is used to limit the data number.
`train_df[(train_df['age'] > 0) & (train_df['age'] < 90)] ` would remove the abnormal value.

File: doc/tutorial/score_card.md (hunk 2 above)
Comment (sneaxiy) [reply doc/tutorial/score_card.md]: It seems that the comment is not resolved.

File: doc/tutorial/score_card.md (hunk 3 above)
Comment (sneaxiy) [reply doc/tutorial/score_card.md]: It seems that the comment is not resolved.

**Review by sneaxiy — APPROVED** [review]
LGTM.

**Pull request merged**